Snakefile

```diff
@@ -1,7 +1,7 @@
 from datetime import date
 from treetime.utils import numeric_date
 
-path_to_fauna = '../fauna'
+path_to_fauna = '../../fauna'
```

Comment:
This will break builds for people who aren't using your file structure. Can you please move `fauna/` and `augur-build/` to be sister directories on your system? This structure is consistent across all Nextstrain builds, eg https://github.com/nextstrain/zika/blob/master/Snakefile#L24